Content.Packaging/ServerPackaging.cs

```diff
@@ -47,6 +47,7 @@ public static class ServerPackaging
         // Python script had Npgsql. though we want Npgsql.dll as well soooo
         "Npgsql",
         "Microsoft",
+        "Discord",
```

Comment:
I wish we had docs on this ngl. Had no idea you had to add this here for nuget packages to show up there.